src/components/CookieBanner/CookieBanner.js

```diff
@@ -0,0 +1,67 @@
+import React, { memo, useCallback, useState, useMemo } from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import checkProps from '@jam3/react-check-extra-props';
+
+import './CookieBanner.scss';
+
+const EXPIRATION_DURATION_MONTHS = 3;
+
+const LOCAL_STORAGE_KEYS = {
+  consent: 'cookie-consent',
+  expiration: 'cookie-expiration'
+};
+
+function CookieBanner({ className }) {
+  const expired = useMemo(() => {
+    let expirationDate = localStorage.getItem(LOCAL_STORAGE_KEYS.expiration);
+    if (expirationDate === null) {
+      return false;
+    }
+
+    expirationDate = new Date(expirationDate);
+    if (expirationDate.getTime() < new Date().getTime()) {
+      return true;
+    } else {
+      return false;
```

Comment:
this can be simplified
`return expirationDate.getTime() < new Date().getTime();`